src/bootstrap/lib.rs

```diff
@@ -1494,8 +1494,12 @@ impl Build {
             {
                 eprintln!(
                     "
-Couldn't find required command: ninja
-You should install ninja, or set `ninja=false` in config.toml in the `[llvm]` section.
+Couldn't find required command: ninja (or ninja-build)
+
+You should install ninja as described at
+<https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages>,
+or set `download-ci-llvm = true` in the `[llvm]` section of `config.toml`
+to download LLVM rather than building it.
 "
```

Comment:
I don't think this last suggestion is great - it might be useful in *addition* to suggesting ninja = false, but it shouldn't replace it altogether.

Reply:
Adding "ninja = false" didn't actually do anything as far as I could see. Is it working properly?

Reply:
It should be working ok.  I think an issue is that if you run `x.py` once with the wrong ninja, then that gets cached in `CMakeCache.txt`, and then if you set `ninja=false`, the cached setup remains.  Did you delete the `llvm` directory after changing the configuration?  If not, then I suspect that is the cause.

Reply:
Yes that's consistent with what I saw. What can we do to improve that?

Reply:
I changed it to include both suggestions.

Reply:
> What can we do to improve that?

Sorry, I'm not too familiar with cmake.  I would maybe look at removing `CMakeCache.txt` at the beginning of the build, but would need to check with the regular llvm developers if that would cause a problem.